lemur/common/celery.py

```diff
@@ -145,10 +145,23 @@ def report_celery_last_success_metrics():
     schedule = current_app.config.get("CELERYBEAT_SCHEDULE")
     for _, t in schedule.items():
         task = t.get("task")
-        last_success = int(red.get(f"{task}.last_success") or 0)
+
+        metric_tags = {}
+        kwargs = t.get("kwargs")
+        if kwargs is not None and "source" in kwargs:
+            source = kwargs["source"]
+            last_success = int(red.get(f"{source}.last_success") or 0)
+            metric_tags["source_name"] = source
+        else:
+            last_success = int(red.get(f"{task}.last_success") or 0)
+
         metrics.send(
-            f"{task}.time_since_last_success", "gauge", current_time - last_success
+            f"{task}.time_since_last_success",
+            "gauge",
+            current_time - last_success,
```

Comment:
If `last_success` is zero, then we may not want to report a metric here at all. Reason being is that `time.time() - 0` would result in a superfluous metric value that would trigger an alert every time a new source is configured. This would deviate from the original implementation, but I think it'd be ok.

Reply:
Alternatively, if we're planning to write the alert rule such that it ignores super large values e.g. >1yr then that's probably ok too and we can leave it as is.

Reply:
Yep will make the change